19.03.5.sh

```diff
@@ -332,6 +335,10 @@ do_install() {
 			dist_version="$(rpm -q --whatprovides ${lsb_dist}-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sed 's/\..*//' | sed 's/Server*//' | sort | tail -1)"
 		;;
 
+		sles)
+			dist_version="$(rpm -q --whatprovides ${lsb_dist}-release --queryformat "%{VERSION}\n" | sed 's/\/.*//' | sort | tail -1)"
+		;;
+
```

Comment:
This value of dist_version is not being used right now by SLES docker install but I left it there because:
1) It would allow the ability to quickly fail out based on the dist_version value if it lacks support (i.e. see centos|fedora|redhat|oraclelinux case)
2) It is consistent with the other OSs offered.

Please let me know if you have any thoughts!

Reply:
As of the latest commit, this value is now being used.